### DISCUSSION TITLE
Is it worth considering the BSP(Build Server Protocol) by Scala and JetBrains?

### DISCUSSION THREAD

**Discussion by koalazub** [helix-editor/helix#10806] [Q&A] [answered]
I just [watched a video on the BSP](https://youtu.be/1Zvf8xbzC3Y?si=PKxq3dHy9IYI6hZy) by the JetBrains and Scala team and thought to ask if this is something that could be interesting for the development of hx? 

It seems like an interesting prospect that could make connections to build systems without so much floundering about. But I don't know too much about the topic other than a top level understanding of it. So I thought I'd ask the question here as to whether it's worth looking at and to leave it to others who are more experienced on the topic

README: https://github.com/build-server-protocol/build-server-protocol?tab=readme-ov-file

**Comment by kirawi** [comment, marked as answer]
See https://github.com/helix-editor/helix/issues/2934
